pilot/pkg/security/authz/builder/testdata/http/allow-full-rule-ambient-in.yaml

```diff
@@ -0,0 +1,76 @@
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: httpbin-1
+  namespace: foo
+spec:
+  selector:
+    matchLabels:
+      app: httpbin
+      version: v1
+  rules:
+    - from:
+        - source:
+            principals: ["principal", "principal-prefix-*", "*-suffix-principal", "*"]
+            requestPrincipals: ["requestPrincipals", "requestPrincipals-prefix-*", "*-suffix-requestPrincipals", "*"]
+            namespaces: ["ns", "ns-prefix-*", "*-ns-suffix", "*"]
+            ipBlocks: ["1.2.3.4", "5.6.0.0/16"]
+            remoteIpBlocks: ["1.2.3.4", "5.6.0.0/16"]
+            notPrincipals: ["not-principal", "not-principal-prefix-*", "*-not-suffix-principal", "*"]
+            notRequestPrincipals: ["not-requestPrincipals", "not-requestPrincipals-prefix-*", "*-not-suffix-requestPrincipals", "*"]
+            notNamespaces: ["not-ns", "not-ns-prefix-*", "*-not-ns-suffix", "*"]
+            notIpBlocks: ["9.0.0.1", "9.2.0.0/16"]
+            notRemoteIpBlocks: ["9.0.0.1", "9.2.0.0/16"]
+      to:
+        - operation:
+            methods: ["method", "method-prefix-*", "*-suffix-method", "*"]
```

Comment:
nit: can you jut put port matches (and maybe 1 other) to reduce noise

Reply:
Yup. Wasn't sure if we should/need to verify all with the ambient flag enabled but I can reduce that to just the destination port rules